pkg/scheduler/apis/config/v1alpha2/defaults.go

```diff
@@ -30,6 +30,11 @@ import (
 	api "k8s.io/kubernetes/pkg/apis/core"
 )
 
+var defaultResourceSpec = []v1alpha2.ResourceSpec{
+	{Name: "cpu", Weight: 1},
+	{Name: "memory", Weight: 1},
```

Comment:
use v1.ResourceCPU and v1.ResourceMemory instead of "cpu" and "memory"?

Reply:
done